src/libraries/tests.proj

```diff
@@ -174,11 +174,17 @@
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Private.Uri/tests/ExtendedFunctionalTests/System.Private.Uri.ExtendedFunctional.Tests.csproj" />
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Reflection.DispatchProxy/tests/System.Reflection.DispatchProxy.Tests.csproj" />
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Security.Cryptography.X509Certificates\tests\System.Security.Cryptography.X509Certificates.Tests.csproj" />
+    
+    <!-- Out of memory https://github.com/dotnet/runtime/issues/66831 -->
+    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Security.Cryptography.Tests\tests\System.Security.Cryptography.Tests.csproj" />
```

Comment:
```suggestion
    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Security.Cryptography\tests\System.Security.Cryptography.Tests.csproj" />
```

Reply:
If we want to disable the test on all Android archetypes, we can put this under L109 ItemGroup
`<ItemGroup Condition="'$(TargetOS)' == 'Android' and '$(RunDisabledAndroidTests)' != 'true'">`

If we want the test to still run on Arm64 since the issue points out Arm,x64,x86, and it looks like the Arm64 lane is still green, we can create a new ItemGroup for Android arm.